freqtrade/exchange/binance.py

```diff
@@ -15,6 +16,8 @@
 
 class Binance(Exchange):
 
+    exchange_name: ExchangeName = ExchangeName.BINANCE
```

Comment:
We do have 2 properties on the exchange for this already - name and id.
Assigning it explicitly like this will be error-prone as it needs manual maintenance as new exchanges are added.

Maybe add a property to the "base" class that returns `self.__class__.__name__` if you really need the exchange class name